### PR TITLE
Game

### PR DESCRIPTION
**Pull request by m-mrcr**
This creates the place_ship_ai and place_ships_user methods, and refactors a few things in other classes, such as the render method in the board class.

### CODE REVIEW

File: lib/cell.rb [hunk 1]
```diff
@@ -34,33 +34,19 @@ def fire_upon
...
-        @status = 'X'
-      else
-        @status = "M"
-      end
-    elsif fired_upon? == false
-      @status = "S"
-    end
-    end
-    @status
+def render(show_ship = false)
```
Comment (teresa-m-knowles) [inline lib/cell.rb]: this is not super common in Ruby but I think looks better in this case!

File: lib/game.rb [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+  attr_reader :computer,
+              :human,
+              :valid
+
+  def initialize
+    @computer = Player.new('computer')
+    @human = Player.new('human')
+  end
+
+  # def welcome
```
Comment (teresa-m-knowles) [inline lib/game.rb]: i commented this out so we could test the game easier

**Comment by m-mrcr** [comment]
> This creates the place_ship_ai and place_ships_user methods, and refactors a few things in other classes, such as the render method in the board class.

edit: cell class***, not board class.

File: lib/game.rb [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+
+  end
+
+  def setup
+    place_ships_ai
+    place_ships_user
+  end
+
+  def place_ships_user
```
Comment (teresa-m-knowles) [inline lib/game.rb]: let's refactor this later to do it the same way we did the place_ships_ai, iterating through each of the ships

**Pull request merged**